usr/share/rear/rescue/default/020_create_skeleton_dirs.sh

```diff
@@ -0,0 +1,34 @@
+# Recreate some directories/symlinks in the rescue system if present.
+# Eventually more stuff from $SHARE_DIR/skel could be moved here.
+# The main use is for symlinks, as their presence in the source tree
+# (under $SHARE_DIR/skel) brings some problems.
+# See https://docs.fedoraproject.org/en-US/packaging-guidelines/Directory_Replacement/
+
+# Other file types can be copied using this mechanism,
+# but it is probably better to use the COPY_AS_IS variable for them.
+# For a directory the difference is that COPY_AS_IS would copy it recursively,
+# while this mechanism recreates the directory as empty.
+
+local skeleton_paths=( /var/run )
+
+local d
+local actual_skeleton_paths=()
+
+# some checks before copying
+for d in "${skeleton_paths[@]}" ; do
+    if ! [ -e "$d" ] ; then
+        Debug "Skeleton path '$d' not present, not creating it under $ROOTFS_DIR"
+        continue
+    fi
+
+    # We could of course replace it, but this should not happen,
+    # so erroring out is safer
+    [ -e "$ROOTFS_DIR/$d" ] && BugError "'$ROOTFS_DIR/$d' already exists - remove '$d' from $SHARE_DIR/skel if it is present there"
+
+    # we want relative paths to avoid superfluous
+    # "Removing leading `/' from member names" message from tar
+    actual_skeleton_paths+=( "./$d" )
+done
+
+# no-recursion because we are interested in the directory itself and not its contents
+tar -f - --no-recursion -C / -c "${actual_skeleton_paths[@]}" | tar -f - -C "$ROOTFS_DIR" -x || Error "Failed to copy some of '${actual_skeleton_paths[*]}' to $ROOTFS_DIR"
```

Comment:
I would appreciate a comment that explains that
because pipefail is not set it is the second 'tar' in the pipe
that determines whether or not the whole operation was successful.

I would appreciate such an explanatory comment
to make it obvious for others (who may at arbitrary later time
have to adapt or enhance things) how it is meant to work, cf. 
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-understand-answer-the-why

Cf. 'pipefail' in usr/share/rear/build/GNU/Linux/100_copy_as_is.sh
In contrast to that I do not understand why we must or can
ignore also here errors in the first 'tar' command.
Is this also intentional here?
If yes, why?

Reply:
@jsmeix good point about `pipefail` - I have not used it, because I took the whole approach from usr/share/rear/rescue/default/010_merge_skeletons.sh and `pipefail` is not used there either, I don't know why.

Reply:
Probably because the code there is ancient? ReaR contains 20 years of development (and learning Bash) history by different people so the most common reason for "why is XYZ over there not done properly" is: Somebody didn't know better (or Bash 3 didn't support it for some array related stuff).

In any case, always feel free to improve things

Reply:
@schlomo fair enough, I will add it here.

Reply:
I added pipefail in last commit [3678f6c](https://github.com/rear/rear/pull/3380/commits/3678f6c321565dbd1d5475a453c20a7f28cc8355)